index.html

```diff
@@ -289,6 +289,113 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <footer>
+        <div class="container-footer">
+          <p class="text-1">
+            I'm always interested in hearing about new projects, so if you'd like
+            to chat please get in
+          </p>
+          <form
+            action=""
+            method="POST"
+            class="form"
+          >
+            <label for="fullname"></label>
+            <input
+              type="text"
+              id="fullname"
+              name="fullname"
+              placeholder="Full name"
+              required
+              maxlength="30"
+            />
+            <label for="email"></label>
+            <input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="name@domain.com"
+              required
+            />
+            <label for="feedback"></label>
+            <textarea
+              rows="5"
+              cols="30"
+              id="feedback"
+              name="feedback"
+              placeholder="Write your message here"
+              required
+              maxlength="505"
+            ></textarea>
```

Comment:
- [ ] As the requirement for this project you need to change the value of max length as required. Please change your textarea to an attribute of maxlength="500". :wink: :wink:

Reply:
It has been updated. Thanks